UPGRADE.md

```diff
@@ -3,6 +3,27 @@ UPGRADE
 
 ## From 0.17 to 0.18.x
 
+### Application
+
+* Moved some of the parameters out of parameters.yml.dist file, please check your configurations
+* Renamed basic parameters to match Symfony Standard's conventions:
+
+Before:
+
+```yaml
+%sylius.database.host%
+%sylius.locale%
+
+# etc.
+```
+
+After:
+
+```yaml
+%database_host%
+%locale%
+```
```

Comment:
I would definitely add a section about HWIOAuth since you've removed both the routing and the firewall config which will break anyone depending on it.

You should highlight the changes in `security.yml` and `routing.yml` telling people to restore these in their own apps if they are using these features.